lib/mobilize_america_client/client/attendances.rb

```diff
@@ -0,0 +1,19 @@
+module MobilizeAmericaClient
+  class Client
+    module Attendances
+      def organization_attendances(organization_id:, updated_since: nil)
+        params = {}
+
+        unless updated_since.nil?
+          params[:updated_since] = updated_since.to_i
+        end
+
+        get(path: "/organizations/#{esc(organization_id)}/attendances", params: params)
+      end
+
+      def event_attendances(organization_id:, event_id:)
```

Comment:
I think this is already callable here:

https://github.com/controlshift/mobilize_america_client/blob/main/lib/mobilize_america_client/client/events.rb#L65